### DISCUSSION TITLE
Offset appears in app injection in the docs

### DISCUSSION THREAD

**Discussion by numpde** [idom-team/idom#921] [Problem]
In this [intro example](https://idom-docs.herokuapp.com/docs/guides/getting-started/index.html#section-1-installing-idom), there is an offset before the word "Sample" that doesn't seem to belong there.

![Screenshot from 2023-02-10 11-18-44](https://user-images.githubusercontent.com/21158052/218027412-57566912-7099-46da-9669-3eae032f5a16.png)

**Comment by rmorshea** [comment]
https://github.com/idom-team/idom/issues/923
